frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -3455,6 +3458,190 @@ def reinterpret(self, x: Conversion, *, bit_width: int) -> Conversion:
         )
         return self.operation(operation, resulting_type, x.result)
 
+    def where(
+        self,
+        resulting_type: ConversionType,
+        condition: Conversion,
+        when_true: Conversion,
+        when_false: Conversion,
+    ) -> Conversion:
+        if condition.original_bit_width != 1 or condition.is_signed:
+            highlights = {
+                condition.origin: "condition is not uint1",
+                self.converting: "but it needs to be for where operation",
+            }
+            self.error(highlights)
+
+        if when_true.is_clear:
```

Comment:
useful in practice, could be added to zero to support it easily.

Reply:
It can be much more optimal though. But I agree it's better to support it. Let me add it and leave a TODO.

Reply:
Ah it can't be done easily because clear integers don't have sign information. Let's leave it out of this PR and create an issue.

Reply:
Mmm looks like it could be a TLU instead yes.